docs/build-clio.md

```diff
@@ -23,9 +23,9 @@ Clio does not require anything other than `compiler.cppstd=20` in your (`~/.cona
 > [!NOTE]
 > Although Clio is built using C++23, it's required to set `compiler.cppstd=20` for the time being as some of Clio's dependencies are not yet capable of building under C++23.
 
-> Mac example:
+Mac example:
```

Comment:
The `>` was making it into a nice block, not sure how it will look without. There may be a better way than `>` though, not sure.

Reply:
We're still creating a nice block, but this text is not part of the block, which in my opinion renders better:

Before: https://github.com/XRPLF/clio/blob/develop/docs/build-clio.md#conan-configuration

After: https://github.com/mathbunnyru/clio/blob/pre_commit_markdownlint/docs/build-clio.md#conan-configuration

Reply:
I meant that the example text is in a block of its own 😄 
Let's ask @maria-robobug to review what looks better and go from there 👍

Reply:
I see no issue with removing the `>` here, also agree it reads better. If you really want to emphasize the text above the block maybe make it bold.

Reply:
Thanks, made it bold as well!